cmd/manager/main.go

```diff
@@ -439,11 +439,13 @@ func startOperator(stopChan <-chan struct{}) error {
 		log.Info("Operator configured to manage all namespaces")
 	case len(managedNamespaces) == 1 && managedNamespaces[0] == operatorNamespace:
 		log.Info("Operator configured to manage a single namespace", "namespace", managedNamespaces[0], "operator_namespace", operatorNamespace)
+		// opts.Namespace implicitly allows watching cluster-scoped resources (e.g. storage classes)
 		opts.Namespace = managedNamespaces[0]
 	default:
 		log.Info("Operator configured to manage multiple namespaces", "namespaces", managedNamespaces, "operator_namespace", operatorNamespace)
-		// the manager cache should always include the operator namespace so that we can work with operator-internal resources
-		opts.NewCache = cache.MultiNamespacedCacheBuilder(append(managedNamespaces, operatorNamespace))
+		// the manager cache should always include the operator namespace so that we can work with operator-internal resources,
+		// and the empty namespace to watch cluster-scoped resources (e.g. storage classes).
+		opts.NewCache = cache.MultiNamespacedCacheBuilder(append(managedNamespaces, operatorNamespace, ""))
```

Comment:
Should this depend on value of `operator.ValidateStorageClassFlag`?

Reply:
Are we caching anything that we don't query for? If not, then I'm fine with leaving as is.

Reply:
We don't, the cache gets built automatically the first time we get or list a particular resource.
(it also means if you list something once you cache it forever)